docs/readthedocs/source/doc/Nano/Overview/pytorch_inference.md

```diff
@@ -227,4 +227,47 @@ InferenceOptimizer.quantize(model,
                             approach='static',
                             max_trials=10,
                             ):
-```
\ No newline at end of file
+```
+
+## Context manager
+BigDL-Nano provides ``InferenceOptimizer.get_context(model=...)`` API to enable automatic context management for PyTorch inference. Only one line of code change, then Nano will automatically provide suitable context management for each accelerated model, it usually contains part of or all of following three types of context manager:
+
+1. torch.no_grad() to disable gradients, which will be used for all model
+   
+2. torch.cpu.amp.autocast(dtype=torch.bfloat16) to run in mixed precision, which will be provided for bf16 related model
+   
+3. torch.set_num_threads() to control thread number, which will be used only if you specify thread_num when `trace/quantize/optimize`
```

Comment:
Should we also put "torch.no_grad()", "torch.cpu.amp.autocast(dtype=torch.bfloat16)", etc., into inline code block ``?

Is it intended to use inline code only for Nano related codes to emphasize them? If not, I feel like we should also add inline code for "torch.no_grad()", etc.